config/initializers/friendly_id.rb

```diff
@@ -1,88 +1,22 @@
-# FriendlyId Global Configuration
-#
-# Use this to set up shared configuration options for your entire application.
-# Any of the configuration options shown here can also be applied to single
-# models by passing arguments to the `friendly_id` class method or defining
-# methods in your model.
-#
-# To learn more, check out the guide:
-#
-# http://norman.github.io/friendly_id/file.Guide.html
+module FriendlyIdOverride
```

Comment:
OK.. I was out of town but will probably have time to make this adjustment today.